src/program.ts

```diff
@@ -121,9 +121,12 @@ export class Program extends DiagnosticEmitter {
 
     this.types = new Map([
       ["i8", Type.i8],
+      ["byte", Type.i8],
       ["i16", Type.i16],
       ["i32", Type.i32],
+      ["int", Type.i32],
       ["i64", Type.i64],
+      ["double", Type.i64],
```

Comment:
in my understanding, a `double` always is a 64-bit float. isn't it?

Reply:
You're right, sorry for sloppy friday night commit ;)
Playing into the hands of AssemblyScript/prototype#104